validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -151,7 +151,10 @@ public static ValidatorClientService create(
                   Optional.empty(),
                   proposerConfigProvider.get(),
                   config.getValidatorConfig().getProposerDefaultFeeRecipient(),
-                  config.getSpec()));
+                  config.getSpec(),
+                  Optional.of(
+                      ValidatorClientService.getKeyManagerPath(services.getDataDirLayout())
+                          .resolve("api_proposer_config"))));
```

Comment:
We've generally use `-` rather than `_` in data paths (e.g. `key-manager`). And since this is a file we probably should add an appropriate extension (.json?)